content/starter-configs/tutorials/wordpress.md

```diff
@@ -48,62 +50,106 @@ define('DB_PASSWORD', 'tugboat');
 define('DB_HOST', 'mysql');
 ```
 
-## Configure Tugboat
+_**🐙 Note:** Make sure your `wp-config.php` is formatted as close to the sample file as possible. The Wordpress CLI
+gets angry when it's not and will release the Kraken on your Tugboat builds._
+
+## Configure Tugboat for Wordpress
 
 The Tugboat configuration is managed by a [YAML file](/setting-up-tugboat/create-a-tugboat-config-file/) at
-.tugboat/config.yml in the git repository. Here's a basic WordPress configuration you can use as a starting point, with
-comments to explain what's going on:
+`.tugboat/config.yml` in the git repository. Copy the code below into that file as a starting point. Then, you'll need
+to tweak a few things for your particular installation.
+
+For each service (php, mysql, etc) Tugboat runs three phases to build your preview:
+[init, update, and build](/building-a-preview/preview-deep-dive/how-previews-work/#the-build-process-explained).
+
+### Phase 1: INIT
+
+In the INIT phase, Tugboat sets up your server and creates the docker containers for your services.
+
+1. If you're using composer to install Wordpress Core, uncomment that line in the `init` phase.
+2. Just after that, make sure that you're mapping the `${DOCROOT}` to whatever you named the directory where you
+   downloaded Wordpress.
```

Comment:
Should item 2 read "If you're not using composer…"? Or perhaps "Whether you are using composer to download Wordpress Core or not, you should symlink `${DOCROOT}` to the directory where Wordpress Core is."

Or something